src/docs2db/database.py

```
@@ -1750,26 +1755,51 @@ async def load_documents(
+            worker_function=load_batch_worker,
+            worker_args=(
+                model,
+                content_dir,
+                host,
+                port,
+                db,
+                user,
+                password,
+                force,
+            ),
+            progress_message=f"Loading files...",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)